src/app/profile/user/details/details.component.scss

```diff
@@ -153,6 +153,7 @@
       font-size: 1.8rem;
       margin-bottom: 1rem;
     }
+    
```

Comment:
Remove this extra line and you're good to go!

Reply:
Removed this line.